docs/cli.md

```diff
@@ -0,0 +1,667 @@
+# Hyperlane CLI
+
+The Hyperlane CLI is a command-line tool written in Typescript that facilitates common operations on Hyperlane, such as deploying the core contracts and/or warp routes to new chains.
```

Comment:
```suggestion
The Hyperlane CLI is a command-line tool written in Typescript that facilitates common operations such as:
- Creating Hyperlane configs.
- Viewing information about core chains.
- Deploying core contracts or warp routes to new chains.
- Sending test messages.
- Checking the status of a message.
```

Gives a broader overview of what the CLI can do before diving into the available commands